### DISCUSSION TITLE
Slack sign up

### DISCUSSION THREAD

**Discussion by camaeel** [bank-vaults/bank-vaults#2433] [Q&A] [answered]
I can't find a way to sign up for slack at https://outshift.slack.com/ 
Is it disabled on purpose?

**Comment by akijakya** [comment]
Hi @camaeel, sorry to hear it didn't work for you, where did you get stuck with the process?

**Reply by ramizpolic** [reply]
We found out that there was an issue with an expired link. Could you confirm @camaeel if you managed to join Slack?

**Reply by camaeel** [reply]
No. When I go to https://outshift.slack.com/ I can only sign-in, there is no sign-up option.

**Reply by lgecse** [reply, marked as answer]
Hi @camaeel , the invite link is under https://outshift.com/slack . We'll update the community links everywhere asap. Thanks for raising this question.
